example_ip_list.txt

```diff
@@ -0,0 +1,3 @@
+192.168.0.2
```

Comment:
- I believe that this file is unnecessary

Reply:
I removed this file